app/Http/Controllers/PhotoController.php

```diff
@@ -440,6 +467,8 @@ public function duplicate(Request $request)
 	}
 
 	/**
+	 * Return a photo as an archive just to annoy @SerenaButler.
```

Comment:
LOL :smiley:. I actually I have it on my TODO list to fix it (I will comment in https://github.com/LycheeOrg/Lychee-front/issues/119).